### ISSUE TITLE
Error reading the data from h5 file format

### ISSUE THREAD

**Issue by archanabhardwaj** [Dana-Farber-AIOS/pathml#316]
I am following tutorial available at https://github.com/Dana-Farber-AIOS/pathml/blob/master/examples/workflow_HE_vignette.ipynb and saved the pre-processing datasets in local computer in h5path format i.e
 
wsi.write("test-pre.h5path"). 

Next I loaded the same TileDataset under dataloader and extracted required information. 

from pathml.ml import TileDataset
from torch.utils.data import DataLoader

dataset = TileDataset("test-pre.h5path")
dataloader = DataLoader(dataset, batch_size = 16, num_workers = 1)

data, masks, hvs, types = next(iter(dataloader))

data variable shows torch.float16 datatype which is causing error for further downstream analysis. 

# few lines of data var
**tensor([[[[212., 213., 215.,  ..., 243., 243., 243.],
          [213., 214., 216.,  ..., 244., 243., 243.],
          [214., 214., 216.,  ..., 244., 244., 243.],**

I worked on few other sample images but still gives me the same error.

Please help me to fix this issue. 

Thanks in advance

**Comment by jacob-rosenthal** [comment]
Hi @archanabhardwaj, this actually sounds like the expected behavior - the h5path file format uses float16 (documented [here](https://pathml.readthedocs.io/en/latest/h5path.html#h5path-file-format)). 

If you need a different dtype for whatever you are doing downstream, maybe you can convert the dtype of the tensor that's returned (https://pytorch.org/docs/stable/generated/torch.Tensor.to.html#torch.Tensor.to)
